sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -727,6 +736,32 @@ Parameters:
 
 Returns: A new executable graph object which can be submitted to a queue.
 
+|
```

Comment:
Did you consider adding this function to a modifiable graph instead (or as well as)? If a user is debugging they might want to dump a graph before, and then after, adding a node. `cudaGraphDebugDotPrint` also takes a modifiable graph parameter, not an executable graph parameter